lib/spack/spack/binary_distribution.py

```diff
@@ -440,10 +438,16 @@ def _fetch_and_cache_index(self, mirror_url, expect_hash=None):
         return True
 
 
+def binary_index_location():
+    """Set up a BinaryCacheIndex for remote buildcache dbs in the user's homedir."""
+    cache_root = os.path.join(misc_cache_location(), 'indices')
```

Comment:
additional `indices` here

Reply:
I have kept this one by removing the extra `indices` subdirectory created in the `BinaryCacheIndex` constructor.